m4_sessions/1708-inning/group_six.md

```diff
@@ -1,53 +1,82 @@
 # Group Six
 
-### Session Title: The not so subtle art of not giving a F#^k
 
-### Session Outcomes: 
+#### Title
+The  (Not-So) Subtle Art of Not Giving a #@!%
+
+#### Description
+
+Release yourself from the prison of meaninglessness and gain full control of your life! You took the first step by coming to Turing, now let's keep the train moving.
+
+Sure, life is chock-full of annoying things, but what if we told you that you can CHOOSE the problems that motivate you, the problems you strive to solve? Take back your power, find your courage and concentrate on what really matters.
+
+1. Embrace the awful
+2. Confront the truth 
+3. Gain elite perspective 
+4. ???
+5. Profit!
+
+### Session Outcomes
 
 Give students the tools to determine for themselves what is important. 
 Worksheet with top five priorities and top five things you don't give a fuck about for personal and professional life
 
 #### Topic Ideas
 * Remove and filter out external pressures and learn to recognize when they affecting your decision making
 
-* The perspective of privilage. Being self aware. Getting worked up over little things compared to real levels of adversity and how it relates to the way we code. Without minifying the personal struggle.
-
 * Not falling into social confitioning traps and understanding the hedonic adaptation and how it affects your hapiness
 
 * Stop wasting your time so you can use it better and feel more fulfilled.
 
 ### Session Outline:
 
-Ice Breaker
-
-Hand out worksheet for priority list and explain
-
-TedTalk
-
-Turn and Talk 2-3 about discussion points
-
-Group talk about those specific discussion points
-
-Topic 1 - Intro
-Topic 1 Group Discussion
-
-Topic 2 - Intro
-Topic 2 Group Discussion
-
-game time
-
-Topic 3 -intro
-Topic 3 Groupd Discussion
-
-Conclusion and Sharing Worksheets(if comfortable)
-
-
+Intro - Will - 5min
+
+Ice Breaker - Adam  - 5min
+
+Hand out worksheet for priority list and explanation - 2min - Lindsay
```

Comment:
If you want this hard copy, send it to me and I'll print it for your session.  Remind students to bring a pen.